cmd/autoperibolossync/main.go

```diff
@@ -144,9 +144,9 @@ func main() {
 	}
 
 	// TODO fix the bumper in upstream to retrieve a default branch
-	defaultBranch := "master"
+	defaultBranch := "main"
```

Comment:
This as an example, main should be equal option in the code, not just replacement

Reply:
That was my bad, now I just swap to check for `main` first and them try `master`.